pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -297,7 +297,8 @@ def make_package(args):
     # Add extra environment variable file into tar-able directory:
     env_vars_tarpath = tempfile.mkdtemp(prefix="p4a-extra-env-")
     with open(os.path.join(env_vars_tarpath, "p4a_env_vars.txt"), "w") as f:
-        f.write("P4A_IS_WINDOWED=" + str(args.window) + "\n")
+        if hasattr(args, 'window'):
```

Comment:
nitpik: Please don't change it, but I just have to say it, we're inconsistent here with single and double quotes

Reply:
yes I see that quote styles are quite mixed 50/50 between single and double quotes ;)